molecule/pki/verify.yml

```diff
@@ -44,8 +44,39 @@
     - name: Populate services facts
       ansible.builtin.service_facts:
 
+    - name: Create log files based on journalctl
+      when: ansible_os_family == 'RedHat'
+      ansible.builtin.shell:
+        cmd: journalctl -u {{ item }} > /var/log/alfresco/{{ item }}.log
+      loop:
+        - alfresco-tengine-aio
+        - alfresco-transform-router
+        - alfresco-shared-fs
+      become: true
+
+    - name: Check in logs a client did connect
+      when: item.edition == acs.edition and ansible_os_family == 'RedHat'
+      ansible.builtin.slurp:
+        src: /var/log/alfresco/{{ item.file }}.log
+      register: ats_log
+      become: true
+      until:
+        - item.pattern in ats_log.content | b64decode
+      retries: 10
+      delay: 3
+      loop:
+        - file: alfresco-tengine-aio
+          pattern: GET Transform Config version
+          edition: Community
+        - file: alfresco-transform-router
+          pattern: GET Transform Config version
+          edition: Enterprise
+        - file: alfresco-shared-fs
+          pattern: TLS virtual host
+          edition: Enterprise
```

Comment:
Decided to add this additional task due:
https://github.com/Alfresco/alfresco-ansible-deployment/actions/runs/9172660358/job/25219680867